azkaban-web-server/src/main/java/azkaban/webapp/servlet/LoginAbstractAzkabanServlet.java

```diff
@@ -556,10 +574,13 @@ private void handleAjaxLoginAction(final String username, final String password,
     if (sessionAdded) {
       ret.put("status", "success");
       ret.put("session.id", session.getSessionId());
+      WebUtils.reportLoginEvent(EventType.USER_LOGIN, username, ip);
     } else {
-      ret.put("error", "Potential DDoS found, the number of sessions for this user and IP "
+      final String message = "Potential DDoS found, the number of sessions for this user and IP "
```

Comment:
Um, I guess I could... but are you sure? 
Concatenation is used 14 times in this file, while `String.format` is not used once. Should'n we be consistent in the techniques that we use to format messages? 
I kind of don't want to change this one string because I did not change any logic here other than introducing a temp var; and I certainly don't want to change all 14 places.

Reply:
sure.